site/content/en/docs/tasks/run/multikueue/job.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Run Kubernetes Job in Multi-Cluster"
+linkTitle: "Kubernetes Job"
+weight: 2
+date: 2024-11-05
+description: >
+  Run a MultiKueue scheduled Kubernetes Job.
+---
+
+## Before you begin
```

Comment:
I would sugger to reorganize this. As intro mention that the Kueue setup will depend on the Kubernetes version used on the cluster. In particular if the `JobManagedBy` feature-gate is enabled on the cluster.

Cluster with JobManagedBy enabled

* Mention that enabling the feature gate is supported only since 1.30 (add a note that is going to be enabled by default in 1.32) 
* In this setup we recommend enabling also `MultiKueueBatchJobWithManagedBy` so that MultiKueue sends life updates reflecting the status of the mirror Job on the worker.

Cluster with JobManagedBy disabled

* For this setup 1.30 isn't required
* In this case `MultiKueueBatchJobWithManagedBy` should be disabled in Kueue. MultiKueue will not perform life updates of the mirror Job from a worker cluster.

Reply:
done